lib/services.rb

```diff
@@ -37,17 +37,11 @@ def self.elasticsearch(cluster: nil, hosts: ENV['ELASTICSEARCH_HOSTS'] || 'http:
   end
 
   def self.statsd_client
-    @statsd_client ||= Statsd.new.tap { |sd| sd.namespace = "govuk.app.search-api" }
+    Statsd.new.tap { |sd| sd.namespace = "govuk.app.search-api" }
   end
 
   def self.cache
-    @cache ||= if ENV['RACK_ENV'] == 'production'
-                 # Using a memory store as this is expected to store signon
-                 # tokens, this will be cleared each time the app starts
-                 ActiveSupport::Cache.lookup_store(:memory_store)
-               else
-                 ActiveSupport::Cache.lookup_store(:null_store)
-               end
+    ActiveSupport::Cache.lookup_store(:memory_store)
```

Comment:
Why are we not using the `null_store` for non-prod environments any more?

Reply:
We now clear the cache before tests. I thought this was sightly easier